app/src/main/res/layout/beer_row.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:weightSum="1"
+    android:orientation="vertical">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="144dp">
+
+        <ImageView
+            android:layout_width="155dp"
+            android:layout_height="143dp"
+            app:srcCompat="@mipmap/ic_launcher"
+            android:id="@+id/ivImage" />
+
+
+        <LinearLayout
+            android:layout_height="74dp"
+            android:layout_width="match_parent"
+            android:orientation="vertical"
+            android:weightSum="1">
+
+
+            <TextView
+                android:id="@+id/tvTitle"
+                android:layout_height="38dp"
+                android:layout_width="wrap_content"
+                android:text="Corona Extra"
```

Comment:
I presume these are left over from debugging. The hard coded strings should be removed now.